tutorial/compiler.md

```diff
@@ -51,3 +51,17 @@ E.g.
 ```shell
 pyccel --compiler=PGI --language=c --export-compile-info=icc.json
 ```
+## Utilising Pyccel within Anaconda Environment
+While Anaconda is a popular way to install Python as it simplifies package management, it can introduce challenges when working with compilers.
+
+Upon installation Anaconda modifies your shell's environment variables.
+
+This can lead to packages installed in conda/anaconda usurping the packages manually installed by the user.
+
+This in turn easily leads to nasty surprises (e.g. using libraries compiled for serial execution instead of optimised parallel implementations.
```

Comment:
Yes I agree, using the one single variable `conda_warnings` looks good, and it will help treat any future conda warnings. ill update the code and the docs. Thank you.

Reply:
fixed: 4291a430ff19c5607300f8d456952450065a1ef5